moped-editor/src/views/projects/projectView/ProjectComponents/TheMap.js

```diff
@@ -103,10 +103,14 @@ export default function TheMap({
     if (shouldShowZoomAlert) {
       errorMessageDispatch({
         type: "show_error",
-        payload: { message: "Zoom in to select features", severity: "error" },
+        payload: {
+          message: "Zoom in to select features",
+          severity: "error",
+          messageType: "componentZoomError",
+        },
       });
     } else {
-      errorMessageDispatch({ type: "hide_error" });
```

Comment:
this is what was hiding the moved component alert -- the useEffect would identify that the zoom alert was not needed, it would dispatch the hide_error action thus hiding the moved component alert.

Reply:
that makes so much sense! 🙌 I like this fix a lot so we are hiding the right alert. It made me start thinking about what a separate alert for the map itself would look like so I wrote up an issue in case it intersects with needs for the projects map some day. https://github.com/cityofaustin/atd-data-tech/issues/17687